federation/test/helpers.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
+import { createTestClient } from 'apollo-server-testing'
+import createServer from '../src/server/createServer'
+import { initialize } from '@dbTools/helpers'
+import { entities } from '@entity/index'
+import { logger } from './testSetup'
+
+export const headerPushMock = jest.fn((t) => {
+  context.token = t.value
+})
+
+const context = {
+  token: '',
+  setHeaders: {
+    push: headerPushMock,
+    forEach: jest.fn(),
+  },
+  clientTimezoneOffset: 0,
```

Comment:
Why? I assume that we ensure that the federation module runs on UTC

Reply:
I don't know the details about this clientTimezoneOffset setting...
Do i have to remove this line or should it be changed to an other value???